upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -776,8 +776,8 @@ func setupNetworking(opt *NewClusterOptions, cluster *api.Cluster) error {
 		if errList := validation.ValidateEtcdVersionForCalicoV3(cluster.Spec.EtcdClusters[0], cluster.Spec.Networking.Calico.MajorVersion, field.NewPath("spec", "networking", "calico")); len(errList) != 0 {
 
 			// This is not a special version but simply of the 3 series
-			for _, etcd := range cluster.Spec.EtcdClusters {
-				etcd.Version = components.DefaultEtcd3Version_1_11
+			for i := range cluster.Spec.EtcdClusters {
+				cluster.Spec.EtcdClusters[i].Version = components.DefaultEtcd3Version_1_11
 			}
 		}
```

Comment:
This should not exist at all. Removing in https://github.com/kubernetes/kops/pull/9803.